js/src/main/scala/org/widok/bindings/HTML.scala

```diff
@@ -79,8 +79,11 @@ object HTML {
 
   case class Button(contents: View*) extends Widget[Button] {
     val rendered = DOM.createElement("button", contents)
+    def buttonType(value: String) = attribute("type", value)
```

Comment:
Avoided it because it is a bit awkward to type, but usually the IDE helps you, so maybe better?